src/util/swrite.cc

```diff
@@ -53,3 +59,42 @@ int swrite( int fd, const char *str, ssize_t len )
 
   return 0;
 }
+
+
+int swrite_timeout( int fd, uint64_t timeout_ms, const char *str, ssize_t len )
```

Comment:
I don't think this (and the code that calls it) is workable.  This is a blocking call inside Mosh.

Mosh is (imperfectly trying to be) an event driven program; it should accept any input and act on it, or generate any output when its file descriptor is ready.  This function blocks the rest of Mosh while waiting to complete the write.

There's an elephant already in the room here, though: Mosh can also block on the tty it's attached to (and block it by not responding fast enough, as well).  I just want to avoid adding another one.